.github/workflows/master_build_merge.yml

```diff
@@ -1496,7 +1496,7 @@ jobs:
             VERSION="$3"
             requiredInputs="1 2 3"
             validateInputs "$requiredInputs" "$@" 
-            ./mvnw package -Dversion=$VERSION
+            ./mvnw package -Drevision=$VERSION
```

Comment:
are you sure all repos uses `revision`. I think its mix of both `revision` and `version`